docs_2.0/manual_EN/4.ngql-guide/16.query-tuning-statements/1.explain-and-profile.md

```diff
@@ -0,0 +1,106 @@
+# EXPLAIN and PROFILE
+
+`EXPLAIN` helps output the execution plan of an nGQL statement without executing the statement, while `PROFILE` executes the statement, then output the execution plan as well as the execution profile. You can optimize the queries for better performance with the execution plan and profile.
+
+## Execution Plan
+
+The execution plan is determined by the nGQL execution planner. nGQL execution planner parses nGQL statements into subtasks and assigns them to several types of plan nodes. 
+
+For example, a simple `SHOW TAGS` statement is parsed as two subtasks and assigned to a `Start` plan node and a `ShowTags` plan node, while a more complex `GO` statement may be parsed as more than 10 subtasks and assigned to plan nodes of the same number.
+
+## Syntax
+
+* `EXPLAIN`
+
+```nGQL
+EXPLAIN [format="dot"] <your nGQL statement>
```

Comment:
`format="dot|row"`, default value is `"row"`

Reply:
Thank you and updated.